tests/scripts/check-generated-files.sh

```diff
@@ -129,6 +129,7 @@ check()
 check scripts/generate_errors.pl library/error.c
 check scripts/generate_query_config.pl programs/test/query_config.c
 check scripts/generate_driver_wrappers.py library/psa_crypto_driver_wrappers.h
+check scripts/generate_driver_wrappers.py  library/psa_crypto_driver_wrappers_no_static.c
```

Comment:
Following up on this 

- `Makefile` is already updated
- `CMakeLists.txt` is already updated
- `make_generated_files.bat` is calling `generate_driver_wrappers.py` so it is ok.

To my understanding we do not have a discrete testing suite for platform/build system intergrity. Personally I am not a big fan of the added complexity added in the flow by using check-generated-files.sh since that looks like functionality that both Cmake and Make have already implemented in a standard way. 

The current dependency flow for those files is a bit convoluted

~~~~~~~
build_system ---> psa_crypto_driver_wrappers_no_static.c --> generate_driver_wrappers.py -> check_generated_files.sh <-- code_style.py
~~~~~~~

So we could re-write or optimise it but that could be a different enhancment outside of the scope of this pr.